test/tests/parallel/set1/spam_virus/undef_spam_subject.bats

```diff
@@ -0,0 +1,63 @@
+load "${REPOSITORY_ROOT}/test/helper/setup"
+load "${REPOSITORY_ROOT}/test/helper/common"
+
+TEST_NAME_PREFIX='Undefined spam subject:'
+
+CONTAINER1_NAME='dms-test-undef_spam_subject-1'
+CONTAINER2_NAME='dms-test-undef_spam_subject-2'
+CONTAINER_NAME=${CONTAINER2_NAME}
+
+function setup() {
+  local CONTAINER_NAME=${CONTAINER1_NAME}
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_AMAVIS=1
+    --env ENABLE_SPAMASSASSIN=1
+    --env SA_SPAM_SUBJECT='undef'
+  )
+  init_with_defaults
+  common_container_setup 'CUSTOM_SETUP_ARGUMENTS'
+  wait_for_smtp_port_in_container "${CONTAINER_NAME}"
+
+  local CONTAINER_NAME=${CONTAINER2_NAME}
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_CLAMAV=1
+    --env SPOOF_PROTECTION=1
+    --env ENABLE_SPAMASSASSIN=1
+    --env REPORT_RECIPIENT=user1@localhost.localdomain
+    --env REPORT_SENDER=report1@mail.my-domain.com
+    --env SA_TAG=-5.0
+    --env SA_TAG2=2.0
+    --env SA_KILL=3.0
+    --env SA_SPAM_SUBJECT="SPAM: "
+    --env VIRUSMAILS_DELETE_DELAY=7
+    --env ENABLE_SRS=1
+    --env SASL_PASSWD="external-domain.com username:password"
```

Comment:
Required to resolve conflict from recent PR merge. This ENV wasn't doing anything to support the test.

```suggestion
```

Reply:
I've resolved this.